angular.json

```diff
@@ -28,7 +28,8 @@
               "node_modules/bootstrap-social/bootstrap-social.css",
               "src/styles.css",
               "src/styles.scss",
-              "src/material.scss"
+              "src/material.scss",
+              "node_modules/material-design-icons/iconfont/material-icons.css"
```

Comment:
What are the contents of the CSS doing? Is it fetching the fonts from a 3rd party site, or from the node_modules? Too lazy to install to find out. :)

Reply:
The CSS references local copies of the fonts, that are added to the project at build-time. Similar to this: 
```
@font-face {
  font-family: 'Material Icons';
  font-style: normal;
  font-weight: 400;
  src: url(https://example.com/MaterialIcons-Regular.eot); /* For IE6-8 */
  src: local('Material Icons'),
    local('MaterialIcons-Regular'),
    url(MaterialIcons-Regular.woff2) format('woff2'),
    url(MaterialIcons-Regular.woff) format('woff'),
    url(MaterialIcons-Regular.ttf) format('truetype');
}
```